app/models/google_event.rb

```diff
@@ -84,9 +78,31 @@ def calendar_service(credentials)
     def client(credentials)
       Signet::OAuth2::Client.new(JSON.parse(credentials))
     end
+
+    def load_emails
+      ConferenceRoom.pluck(:email)
+    end
+
+    def new_time_low(time)
+      if time > time.beginning_of_hour + 30.minutes
+        time.beginning_of_hour + 30.minutes
+      else
+        time.beginning_of_hour
+      end
+    end
+
+    def new_time_high(time)
```

Comment:
This is for rendering purposes.
View requires events to have 30 minutes time granularity.